mros1_reasoner/launch/run.launch

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="onto" default="abb_dualarm_mm_complete.owl"/>
+  <arg name="onto" default="kb.owl"/>
```

Comment:
I still get confused by this `onto` ("what is this *onto* on?").

Out-of-scope for this PR, but I'd suggest to rename to `ontology`.

Reply:
Created issue #87